rom/dev/src/flow/cold_reset/ldev_id.rs

```diff
@@ -179,6 +184,7 @@ impl LocalDevIdLayer {
             None,
             stable_idev,
             HmacMode::Hmac512,
+            KeyUsage::default().set_aes_key_en(),
```

Comment:
Should this enable `set_hmac_key_en` as well for deriving DOT and LOCK stable identity keys? Similar question for `derive_stable_identity_root_ldev` below

Reply:
No, it's not necessary.

The goal for these keys is to derive firmware-visible keys through using AES CMAC -> HMAC. So they will always go through AES CMAC first so that the result is visible to firmware.

We won't be deriving any other key vault keys off of them.